include/seqan3/alignment/configuration/utility.hpp

```diff
@@ -165,8 +167,9 @@ inline constexpr align_cfg::id align_config_type_to_id_v = align_config_type_to_
 inline constexpr std::array<std::array<bool, static_cast<uint8_t>(align_cfg::id::SIZE)>,
                             static_cast<uint8_t>(align_cfg::id::SIZE)> align_config_validation_matrix =
 {
-    // gap
-    { false }  // gap
+    // gap    global
+    { { false, true },// gap
```

Comment:
This should be different. See: 
https://github.com/seqan/seqan3/pull/370/files#diff-7712d783236ca2bf66565fa3c8801c47R171
```
{ false, true,
  true, false }
```